packages/stable/src/__tests__/api/documentsApi.int.spec.ts

```diff
@@ -3,12 +3,34 @@
 import CogniteClient from '../../cogniteClient';
 import { setupLoggedInClient } from '../testUtils';
 import { DocumentSearchResponse } from '@cognite/sdk-stable/dist';
+import { TextEncoder } from 'util';
 
 describe('Documents integration test', () => {
   let client: CogniteClient;
+  let fileId: number;
 
   beforeAll(async () => {
     client = setupLoggedInClient();
+
+    // ensure we have a file for testing
+    try {
+      const resp = await client.documents.list({
+        limit: 1,
+      });
+      fileId = resp.items[0].id;
+    } catch (error) {
+      const fileContent = new TextEncoder().encode('test data');
+      const file = await client.files.upload(
+        {
+          name: 'test.txt',
+          mimeType: 'text/plain',
+        },
+        fileContent,
+        false,
+        true
+      );
+      fileId = file.id;
```

Comment:
Can we make this loop until the document is actually available? The current way this will work is that first time (for an empty test project) the tests will unexpectedly fail, and it will not be clear why it fails, while it is just waiting for the pipeline to process the file. Then "suddenly" it will start working again. We should avoid such behaviour in our tests.